tests/e2e/x/transfer/virtuous.go

```diff
@@ -76,7 +79,39 @@ var _ = e2e.DescribeXChainSerial("[Virtuous Transfer Tx AVAX]", func() {
 
 			// Ensure the same set of 10 keys is used for all tests
 			// by retrieving them outside of runFunc.
-			testKeys := env.AllocatePreFundedKeys(10)
+			testKeys := []*secp256k1.PrivateKey{
+				// The funded key will be the source of funds for the new keys
+				env.PreFundedKey,
+			}
+			newKeys, err := tmpnet.NewPrivateKeys(9)
+			require.NoError(err)
+			testKeys = append(testKeys, newKeys...)
+
+			const transferPerRound = units.MilliAvax
+
+			tc.By("Funding new keys")
+			fundingWallet := e2e.NewWallet(tc, env.NewKeychain(), env.GetRandomNodeURI())
+			for _, key := range newKeys {
+				_, err := fundingWallet.X().IssueBaseTx(
+					[]*avax.TransferableOutput{{
+						Asset: avax.Asset{
+							ID: fundingWallet.X().Builder().Context().AVAXAssetID,
+						},
+						Out: &secp256k1fx.TransferOutput{
+							// Enough for 1 transfer per round
+							Amt: totalRounds * transferPerRound,
+							OutputOwners: secp256k1fx.OutputOwners{
+								Threshold: 1,
+								Addrs: []ids.ShortID{
+									key.Address(),
+								},
+							},
+						},
+					}},
+					tc.WithDefaultContext(),
+				)
+				require.NoError(err)
+			}
 
 			runFunc := func(round int) {
```

Comment:
Note to myself: This test looks... Interesting... Should look to see what the use actually is.